test/connectors/TrustsConnectorSpec.scala

```diff
@@ -383,5 +387,59 @@ class TrustsConnectorSpec extends SpecBase with ScalaFutures
       application.stop()
     }
 
+    "getTrustName" in {
+
+      val trustName = "Trust Name"
+
+      val json = JsString(trustName)
+
+      val application = applicationBuilder()
+        .configure(
+          Seq(
+            "microservice.services.trusts.port" -> server.port(),
+            "auditing.enabled" -> false
+          ): _*
+        ).build()
+
+      val connector = application.injector.instanceOf[TrustsConnector]
+
+      server.stubFor(
+        get(urlEqualTo(getTrustNameUrl(identifier)))
+          .willReturn(okJson(json.toString))
+      )
+
+      val result = connector.getTrustName(identifier)
+
+      whenReady(result) { r =>
+        r mustBe trustName
+      }
+
+      application.stop()
+    }
+
+    "removeTrustTypeDependentTransforms" in {
+
+      val application = applicationBuilder()
+        .configure(
+          Seq(
+            "microservice.services.trusts.port" -> server.port(),
+            "auditing.enabled" -> false
+          ): _*
+        ).build()
+
+      val connector = application.injector.instanceOf[TrustsConnector]
+
+      server.stubFor(
+        delete(urlEqualTo(removeTrustTypeDependentTransformsUrl(identifier)))
+          .willReturn(ok)
+      )
+
+      val result = connector.removeTrustTypeDependentTransforms(identifier)
+
+      result.futureValue.status mustBe OK
```

Comment:
This should be using whenReady as discovered last week that it sometimes doesn't assert in time.